### PR TITLE
Set up Hugo with documentation stubs

### PR DESCRIPTION
**Pull request by anormananderson**
Not sure if there is a way to combine this with #12, but this is the corresponding pull request I made.

### CODE REVIEW

**Comment by anormananderson** [comment]
@callumcurtis this should be good to go - only unknown is whether the workflow file will run correctly. If you'd rather I could open a separate repo to check first?

**Comment by callumcurtis** [comment]
It's low risk so let's give it a shot first and then you can add a fix on top if something goes wrong.

File: .github/workflows/hugo.yaml [hunk 1]
```diff
@@ -0,0 +1,81 @@
+# Sample workflow for building and deploying a Hugo site to GitHub Pages
+name: Deploy Hugo site to Pages
```
Comment (callumcurtis) [inline .github/workflows/hugo.yaml:2]:
```suggestion
name: Deploy Hugo Site to Pages
```

File: docs/hugo.toml [hunk 2]
```diff
@@ -0,0 +1,14 @@
+baseURL = 'https://restauwants.github.io/restauwants/'
+languageCode = 'en-us'
+title = 'Restauwants Documentation'
```
Comment (callumcurtis) [inline docs/hugo.toml:3]:
```suggestion
title = 'RestauWants Documentation'
```

File: docs/hugo.toml [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+
+[module]
+[[module.imports]]
+    path = 'github.com/McShelby/hugo-theme-relearn'
+
+# Change the default theme to be use when building the site with Hugo
+theme = "hugo-theme-relearn"
```
Comment (callumcurtis) [inline docs/hugo.toml:10]: Can we change the theming to match our colors/logo?

**Review by callumcurtis — CHANGES_REQUESTED** [review]
Couple of nitpicks, otherwise LGTM. 👍

**Review by callumcurtis — APPROVED** [review]

**Comment by callumcurtis** [comment]
Haven't pulled to look at the theming locally but we can iterate on that as we go. For the future, a screenshot of the theme appearance in the browser would be helpful! LGTM

**Pull request merged**